File: Auction_Project/Models/Items/ItemResponseForAdminDTO.cs
using Auction_Project.Models.Bids;

namespace Auction_Project.Models.Items
{
    public class ItemResponseForAdminDTO
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string? Desc { get; set; }

        public decimal? InitialPrice { get; set; }

        public DateTime? endTime { get; set; }

        public List<int>? Gallery { get; set; }

        public List<Bid>? BidsOnItem { get; set; }

    }
}

File: Frontend/src/app/api/models/item-response-for-admin-dto.ts
/* tslint:disable */
/* eslint-disable */
import { BidResponseForAdminDto } from './bid-response-for-admin-dto';
export interface ItemResponseForAdminDto {
  bidsOnItem?: null | Array<BidResponseForAdminDto>;
  desc?: null | string;
  endTime?: null | number;
  gallery?: null | Array<number>;
  id?: number;
  initialPrice?: null | number;
  name?: null | string;
}
